build.psm1

```diff
@@ -137,6 +137,25 @@ function Copy-CompatibilityProfiles
     Copy-Item -Force $profileDir/* $targetProfileDir
 }
 
+function Start-ReferenceBuild
+{
+    try {
+        Push-Location Reference
+        $buildOutput  = dotnet build --configuration Release
+        if ( ! $? ) {
+            throw $buildOutput
+        }
+        $buildOutput += dotnet pack --configuration Release
+        if ( ! $? ) {
+            throw $buildOutput
+        }
+        Copy-Item bin/Release/Microsoft.Windows.PowerShell.ScriptAnalyzer.1.18.4.nupkg .
```

Comment:
`Copy-Item` supports wildcards:
```suggestion
        Copy-Item bin/Release/Microsoft.Windows.PowerShell.ScriptAnalyzer.*.nupkg .
```

Reply:
it's sort of a test to be sure the right version was created, but this will be fine